tests/image_builder/monitors/base/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
```

Comment:
As discussed in the past, would it perhaps make sense to utilize single image for all the monitor tests (we would simply bundle in all the dependencies we need)?

This would probably make it a bit easier to keep up to date. Since maintaining that many Dockerfile and related files can get quite messy.

Reply:
This way we would also avoid potential issues with single Circle CI job cache growing very large (since I assume we will run all the monitors tests as part of the same job, otherwise it would get to unwindy).